example/docker-compose.yml

```diff
@@ -29,27 +31,33 @@ services:
       - .:/conf
       - logs:/home/envoy/logs:rw
     ports:
-      - 8080:80
-  wasm-logs:
+      - 8080:8080
+  envoy-logs:
     depends_on:
       - envoy
+      - wasm-logs
     image: debian:11-slim
     entrypoint: bash
     command:
       - -c
-      - tail -c +0 -f /home/envoy/logs/envoy.log | grep --line-buffered "[critical][wasm]" | tee /home/envoy/logs/ftw.log
+      - tail -c +0 -f /home/envoy/logs/envoy.log
     volumes:
-      - logs:/home/envoy/logs:rw
-  envoy-logs:
+      - logs:/home/envoy/logs:ro
+  wasm-logs:
     depends_on:
       - envoy
-      - wasm-logs
     image: debian:11-slim
     entrypoint: bash
     command:
       - -c
-      - tail -c +0 -f /home/envoy/logs/envoy.log
+      - tail -c +0 -f /home/envoy/logs/envoy.log | grep --line-buffered "[critical][wasm]"
     volumes:
       - logs:/home/envoy/logs:ro
+  tests:
+    depends_on:
+      - envoy
+    build:
+      context: ../e2e/
```

Comment:
circular dependency here. I see e2e depending on example and example depending on e2e. I think what you want here is to add a new docker-compose in the e2e and depend on this one? https://stackoverflow.com/questions/55650342/import-docker-compose-file-in-another-compose-file